core/templates/dev/head/css/oppia.css

```diff
@@ -119,6 +119,78 @@ body {
   margin-top: 15px;
 }
 
+.oppia-profile-tabs md-content {
+  background-color: white !important; 
+}
+
+.oppia-profile-tabs md-content md-tabs {
+  background: #f6f6f6;
+  border: 1px solid #e1e1e1; 
+}
+
+.oppia-profile-tabs md-content md-tabs md-tabs-wrapper {
+  background: white; 
+}
+
+.oppia-profile-tabs md-content h1:first-child {
+  margin-top: 0; 
+}
+
+.oppia-profile-tabs md-tab {
+  background-color: #009688;
+}
+
+.oppia-profile-tabs md-tab-ink-bar {
+  background-color: black;
+}
+
+md-tabs. md-padding {
+  width: 20%;
+}
+
+.oppia-profile-tabs md-tab-label {
+    color: #ffffff;
+}
+
+.stat-container-top {
```

Comment:
Use class names prefixed with .oppia- to indicate that these originate from oppia (and not third-party libs).